promql/engine.go

```diff
@@ -1332,6 +1350,9 @@ func (ev *evaluator) eval(expr parser.Expr) parser.Value {
 
 		res := newEv.eval(e.Expr)
 		ev.currentSamples = newEv.currentSamples
+		// Only update the peak samples if result evaluated by the new evaluator is
```

Comment:
This comment doesn't seem necessary, as that's handled inside the function.

Reply:
Sure, will remove it.